apps/angular/src/app/features/anime-dashboard/components/anime-table/anime-table.component.ts

```diff
@@ -12,11 +14,14 @@ import { Anime } from '@js-camp/core/models/anime';
 	templateUrl: './anime-table.component.html',
 	styleUrl: './anime-table.component.css',
 	changeDetection: ChangeDetectionStrategy.OnPush,
-	imports: [CommonModule, MatTableModule, EmptyPipe, BasicProgressSpinnerComponent],
+	imports: [CommonModule, MatTableModule, EmptyPipe, MatSortModule, BasicProgressSpinnerComponent],
 })
 export class AnimeTableComponent {
+
 	/** Anime list. */
-	@Input() public animeList: Anime[] | null = null;
+	@Input() public animeList: readonly Anime[] | null = [];
```

Comment:
```suggestion
	@Input() 
	public animeList: readonly Anime[] = [];
```

Why might the value be `null`? Why does this component even care that the value might be different from the entity the component is working with?

Leave the data handling to the consumers of this component.